lib/vagrant-openshift/action/try_restart_openshift3.rb

```diff
@@ -26,7 +26,7 @@ def initialize(app, env)
         end
 
         def call(env)
-          sudo(env[:machine], "systemctl try-restart openshift")
+          sudo(env[:machine], "systemctl restart openshift")
```

Comment:
So my issue, then, is that in order to install a router I need openshift to be running.  Since we were shooting for the all-in-one experience I forced it to come up rather than ignore if it wasn't started.  Should I just let the router install fail if it is not up?